tests/sles4sap/netweaver_test_instance.pm

```diff
@@ -25,7 +25,10 @@ sub run {
 
     # The SAP Admin was set in sles4sap/netweaver_install
     $self->set_sap_info(get_required_var('INSTANCE_SID'), get_required_var('INSTANCE_ID'));
-    $self->test_pids_max;
+    # Don't test pids_max on migration
+    if (!get_var('UPGRADE')) {
+        $self->test_pids_max;
+    }
```

Comment:
I like this more as either `$self->test_pids_max unless get_var('UPGRADE');` or `$self->test_pids_max if (!get_var('UPGRADE'));`. Did you test the change?

Reply:
Fixed.